client/signup/steps/woocommerce-install/step-business-info/index.tsx

```diff
@@ -0,0 +1,229 @@
+import { SelectControl, TextControl } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { ReactElement } from 'react';
+import { useSelector } from 'react-redux';
+import { LoadingEllipsis } from 'calypso/components/loading-ellipsis';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import SupportCard from '../components/support-card';
+import { ActionSection, StyledNextButton } from '../confirm';
+import {
+	useSiteSettings,
+	WOOCOMMERCE_DEFAULT_COUNTRY,
+	WOOCOMMERCE_ONBOARDING_PROFILE,
+} from '../hooks/use-site-settings';
+import { getRevenueOptions } from './revenue-options';
+import type { WooCommerceInstallProps } from '..';
+import './style.scss';
+
+export default function StepBusinessInfo( props: WooCommerceInstallProps ): ReactElement | null {
+	const { goToStep, isReskinned } = props;
+	const { __ } = useI18n();
+
+	const siteId = useSelector( getSelectedSiteId ) as number;
+	const currencyCode = useSelector( getCurrentUserCurrencyCode ) as string;
+
+	const { get, save, update } = useSiteSettings( siteId );
+
+	function updateProductTypes( type: string ) {
+		updateOnboardingProfile( 'product_type', type );
+	}
+
+	function updateProductCount( count: string ) {
+		updateOnboardingProfile( 'product_count', count );
+	}
+
+	function updateSellingVenues( venue: string ) {
+		updateOnboardingProfile( 'selling_venues', venue );
+	}
+
+	function updateOtherPlatform( platform: string ) {
+		updateOnboardingProfile( 'other_platform', platform );
+	}
+
+	function updateOtherPlatformName( name: string ) {
+		updateOnboardingProfile( 'other_platform_name', name );
+	}
+
+	function updateOnboardingProfile( key: string, value: string | boolean ) {
+		const profile_data = get( WOOCOMMERCE_ONBOARDING_PROFILE ) || {};
+
+		profile_data[ key ] = value;
+
+		update( WOOCOMMERCE_ONBOARDING_PROFILE, profile_data );
+	}
+
+	function getContent() {
+		const profile_data = get( WOOCOMMERCE_ONBOARDING_PROFILE );
+
+		return (
+			<>
+				<div className="step-business-info__info-section" />
+				<div className="step-business-info__instructions-container">
+					<SelectControl
+						label={ __( 'What type of products will be listed?', 'woocommerce-admin' ) }
+						value={ profile_data?.product_types?.shift() }
+						options={ [
+							{ value: '', label: '' },
+							{ value: 'physical', label: __( 'Physical Products', 'woocommerce-admin' ) },
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **59** times:
`translate( 'Physical products' )` *ES Score: 12*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/product-types-step)